DPL: improve logs for new calibration scheme

ktf opened the pull request:
- Report correct timeout for readers.
- Improve message when dropping data which is not going to be processed by
  the CalibrationOnly mode.

davidrohr commented on Framework/Core/src/DataRelayer.cxx:
```diff
@@ -335,7 +337,11 @@ void DataRelayer::setOldestPossibleInput(TimesliceId proposed, ChannelIndex chan
           auto& state = mContext.get<DeviceState>();
           if (state.transitionHandling != TransitionHandlingState::NoTransition && DefaultsHelpers::onlineDeploymentMode()) {
-            LOGP(warning, "Missing {} (lifetime:{}) while dropping incomplete data in slot {} with timestamp {} < {}.", DataSpecUtils::describe(input), input.lifetime, si, timestamp.value, newOldest.timeslice.value);
+            if (state.allowedProcessing == DeviceState::CalibrationOnly) {
```
In case we are in calibration mode, I would drop the MISSING message completely. It is expected to be missing, so no need to inform about it.
And then for the "Dropping lifetime::timeframe" warning, I would demote to info in calibration mode and pring " [INFO]: Not processing in calibration mode:...."

davidrohr commented on Framework/Core/src/DataProcessingDevice.cxx:
```diff
@@ -1395,10 +1395,11 @@ void DataProcessingDevice::Run()
+            O2_SIGNPOST_EVENT_EMIT_INFO(device, lid, "run_loop", "New state requested. Waiting for %d seconds before quitting.", timeout);
           } else {
-            O2_SIGNPOST_EVENT_EMIT_INFO(device, lid, "run_loop", "New state requested. Waiting for %d seconds before switching to READY state.", (int)deviceContext.exitTransitionTimeout);
+            O2_SIGNPOST_EVENT_EMIT_INFO(device, lid, "run_loop", "New state requested. Waiting for %d seconds before switching to READY state.", timeout);
```
Could we also add (as I wrote in https://its.cern.ch/jira/browse/O2-4308): For source devices like the readout-proxy, I would write "Waiting for DATAPROCESSING_TIMEOUT seconds before dropping the remaining input".

alibuild commented:
Error while checking build/O2/fullCI_slc9 for 9610896435a0150cdd7cb75d1a784b209e10d59b at 2025-06-25 14:59:
```
[3703/5748] Linking CXX executable stage/bin/o2-calibration-mch-dcs-sim-workflow
[3704/5748] Building CXX object Detectors/EMCAL/calibration/CMakeFiles/O2test-emcal-PedestalProcessorData.dir/test/testPedestalProcessorData.cxx.o
[3705/5748] Linking CXX executable stage/bin/o2-calibration-mid-dcs-sim-workflow
[3706/5748] Linking CXX shared library stage/lib64/libO2FV0DCSMonitoring.so
[3707/5748] Building CXX object Detectors/ITSMFT/MFT/condition/CMakeFiles/O2lib-MFTCondition.dir/src/DCSConfigUtils.cxx.o
[3708/5748] Building CXX object Detectors/MUON/MCH/Conditions/CMakeFiles/O2exe-mch-scan-hvlv-ccdb.dir/__/__/__/__/Common/Utils/src/fpu.cxx.o
[3709/5748] Linking CXX shared library stage/lib64/libO2FDDDCSMonitoring.so
[3710/5748] Building CXX object Detectors/MUON/MCH/IO/CMakeFiles/O2exe-mch-statusmaps-reader-workflow.dir/__/__/__/__/Common/Utils/src/fpu.cxx.o
[3711/5748] Building CXX object Detectors/EMCAL/calibration/CMakeFiles/O2lib-EMCALCalibration.dir/src/EMCALPedestalHelper.cxx.o
[3712/5748] Linking CXX executable stage/tests/o2-test-ctf-ft0
[3713/5748] Building CXX object Detectors/MUON/MCH/IO/CMakeFiles/O2exe-mch-statusmaps-writer-workflow.dir/__/__/__/__/Common/Utils/src/fpu.cxx.o
[3714/5748] Building CXX object Detectors/FIT/FV0/dcsmonitoring/CMakeFiles/O2exe-fv0-dcs-workflow.dir/workflow/fv0-dcs-data-workflow.cxx.o
[3715/5748] Building CXX object Detectors/EMCAL/calibration/CMakeFiles/O2lib-EMCALCalibration.dir/src/EMCDCSProcessor.cxx.o
[3716/5748] Linking CXX executable stage/bin/o2-fv0-dcs-workflow
[3717/5748] Building CXX object Detectors/ITSMFT/MFT/condition/CMakeFiles/O2lib-MFTCondition.dir/src/MFTDCSProcessor.cxx.o
[3718/5748] Generating G__O2FT0DCSMonitoring.cxx, ../../../../stage/lib64/G__O2FT0DCSMonitoring_rdict.pcm, ../../../../stage/lib64/libO2FT0DCSMonitoring.rootmap
[3719/5748] Building CXX object Detectors/FIT/FV0/dcsmonitoring/CMakeFiles/O2exe-fv0-dcs-config-workflow.dir/workflow/fv0-dcs-config-workflow.cxx.o
[3720/5748] Building CXX object Detectors/EMCAL/calibration/CMakeFiles/O2lib-EMCALCalibration.dir/src/EMCALTimeCalibData.cxx.o
[3721/5748] Building CXX object Detectors/FIT/FDD/dcsmonitoring/CMakeFiles/O2exe-fdd-dcs-workflow.dir/workflow/fdd-dcs-data-workflow.cxx.o
[3722/5748] Building CXX object Detectors/MUON/MCH/Status/CMakeFiles/O2lib-MCHStatus.dir/src/StatusMapCreatorParam.cxx.o
[3723/5748] Building CXX object Detectors/EMCAL/calibration/CMakeFiles/O2lib-EMCALCalibration.dir/src/PedestalProcessorDevice.cxx.o
[3724/5748] Building CXX object Detectors/FIT/FT0/dcsmonitoring/CMakeFiles/O2exe-ft0-dcs-workflow.dir/__/__/__/__/Common/Utils/src/fpu.cxx.o
[3725/5748] Building CXX object Detectors/FIT/FT0/dcsmonitoring/CMakeFiles/O2exe-ft0-dcs-config-workflow.dir/__/__/__/__/Common/Utils/src/fpu.cxx.o
[3726/5748] Building CXX object Detectors/DCS/testWorkflow/CMakeFiles/O2exe-dcs-proxy.dir/src/dcs-proxy.cxx.o
[3727/5748] Building CXX object Detectors/MUON/MCH/Status/CMakeFiles/O2lib-MCHStatus.dir/src/StatusMap.cxx.o
[3728/5748] Building CXX object Detectors/MUON/MCH/Status/CMakeFiles/O2exe-mch-statusmap-creator-workflow.dir/__/__/__/__/Common/Utils/src/fpu.cxx.o
[3729/5748] Building CXX object Detectors/FIT/FDD/dcsmonitoring/CMakeFiles/O2exe-fdd-dcs-config-workflow.dir/workflow/fdd-dcs-config-workflow.cxx.o
[3730/5748] Building CXX object Detectors/ITSMFT/MFT/condition/CMakeFiles/O2lib-MFTCondition.dir/src/DCSConfigReader.cxx.o
[3731/5748] Building CXX object Detectors/MUON/MCH/Status/CMakeFiles/O2exe-mch-statusmap-to-rejectlist.dir/__/__/__/__/Common/Utils/src/fpu.cxx.o
[3732/5748] Linking CXX executable stage/bin/o2-fv0-dcs-config-workflow
[3733/5748] Building CXX object Detectors/MUON/MCH/Status/CMakeFiles/O2lib-MCHStatus.dir/src/HVStatusCreator.cxx.o
[3734/5748] Linking CXX executable stage/bin/o2-fdd-dcs-workflow
[3735/5748] Building CXX object Detectors/EMCAL/calibration/CMakeFiles/O2lib-EMCALCalibration.dir/src/EMCALCalibExtractor.cxx.o
[3736/5748] Building CXX object Detectors/MUON/MCH/Status/CMakeFiles/O2test-mch-statusmap.dir/__/__/__/__/Common/Utils/src/fpu.cxx.o
[3737/5748] Building CXX object Detectors/MUON/MCH/Status/CMakeFiles/O2lib-MCHStatus.dir/G__O2MCHStatus.cxx.o
[3738/5748] Building CXX object Detectors/EMCAL/calibration/CMakeFiles/O2exe-calibration-emcal-pedestal-processor-workflow.dir/testWorkflow/emc-pedestal-processor-workflow.cxx.o
[3739/5748] Building CXX object Detectors/MUON/MID/Simulation/CMakeFiles/O2lib-MIDSimulation.dir/src/ChamberResponseParams.cxx.o
[3740/5748] Linking CXX executable stage/bin/o2-fdd-dcs-config-workflow
[3741/5748] Building CXX object Detectors/EMCAL/calibration/CMakeFiles/O2lib-EMCALCalibration.dir/src/PedestalCalibDevice.cxx.o
[3742/5748] Building CXX object Detectors/MUON/MID/Simulation/CMakeFiles/O2lib-MIDSimulation.dir/src/ChamberEfficiencyResponse.cxx.o
[3743/5748] Building CXX object Detectors/EMCAL/calibration/CMakeFiles/O2lib-EMCALCalibration.dir/src/EMCALChannelData.cxx.o
[3744/5748] Building CXX object Detectors/FIT/FT0/dcsmonitoring/CMakeFiles/O2lib-FT0DCSMonitoring.dir/G__O2FT0DCSMonitoring.cxx.o
[3745/5748] Building CXX object Detectors/MUON/MID/Simulation/CMakeFiles/O2lib-MIDSimulation.dir/src/ClusterLabeler.cxx.o
[3746/5748] Building CXX object Detectors/EMCAL/calibration/CMakeFiles/O2exe-calibration-emcal-pedestal-calib-workflow.dir/testWorkflow/emc-pedestal-calib-workflow.cxx.o
[3747/5748] Building CXX object Detectors/MUON/MID/Simulation/CMakeFiles/O2lib-MIDSimulation.dir/src/ChamberResponse.cxx.o
[3748/5748] Building CXX object Detectors/ITSMFT/MFT/condition/CMakeFiles/O2lib-MFTCondition.dir/G__O2MFTCondition.cxx.o
[3749/5748] Building CXX object Detectors/MUON/MID/Simulation/CMakeFiles/O2lib-MIDSimulation.dir/src/ChamberHV.cxx.o
[3750/5748] Building CXX object Detectors/MUON/MCH/IO/CMakeFiles/O2exe-mch-statusmaps-reader-workflow.dir/src/StatusMapReaderSpec.cxx.o
[3751/5748] Building CXX object Detectors/FIT/FT0/dcsmonitoring/CMakeFiles/O2lib-FT0DCSMonitoring.dir/src/FT0DCSDataProcessor.cxx.o
[3752/5748] Building CXX object Detectors/MUON/MID/Simulation/test/CMakeFiles/O2test-mid-Geometry.dir/__/__/__/__/__/Common/Utils/src/fpu.cxx.o
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/14030/9610896435a0150cdd7cb75d1a784b209e10d59b/build_O2_fullCI_slc9/pretty.html).

davidrohr commented:
@ktf : What is the status here? Could you address my comments as far as possible, and additional messages we don't have in here yet, we could bring in with a separate PR?

ktf committed:
DPL: improve logs for new calibration scheme

- Report correct timeout for readers.
- Improve message when dropping data which is not going to be processed by
  the CalibrationOnly mode.

ktf commented:
I have just updated the with my understanding of your comments.

davidrohr reviewed (approved):
Great, without actually trying, this looks exactly like what I have asked for. Thx!
